src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -210,7 +210,6 @@ func (q *qemuAmd64) enableProtection() error {
 			q.qemuMachine.Options += ","
 		}
 		q.qemuMachine.Options += "kvm-type=tdx,confidential-guest-support=tdx"
-		q.kernelParams = append(q.kernelParams, Param{"tdx_guest", ""})
```

Comment:
is this `TDX_GUEST` kernel configuration already upstream? More precisely, any risk of that change in near future so that we will need to revert this change?

Reply:
AFAIR it's not upstreamed yet, but the `tdx_guest` parameter was from an older review.  So, if it changes, hardly it'll change back to that format. :-)